backend/api/Services/InspectionService.cs

```diff
@@ -110,7 +113,7 @@ private IQueryable<Inspection> GetInspections(bool readOnly = true)
             return inspection;
         }
 
-        public async Task<IDAInspectionDataResponse?> GetInspectionStorageInfo(string inspectionId)
+        private async Task<IDAInspectionDataResponse?> GetInspectionStorageInfo(string inspectionId)
```

Comment:
For it to work in prod we would need change CallApiForUserAsync to CallApiForAppAsync below

Reply:
We could in that case have an environment check here and use the correct function based on that. Then we can test it by changing the local environment to prod before merging it

Reply:
I think it should be CallApiForApp for all environments